great_expectations/render/renderer/slack_renderer.py

```diff
@@ -175,3 +194,39 @@ def _get_report_element(self, docs_link):
                 "No docs link found. Skipping data docs link in Slack message."
             )
         return report_element
+
+    def create_failed_expectations_text(self, validation_results: List[dict]) -> str:
+        failed_expectations_str = "\n*Failed Expectations*:\n"
+        for expectation in validation_results:
+            if not expectation["success"]:
+                expectation_name = expectation["expectation_config"]["expectation_type"]
+                expectation_kwargs = expectation["expectation_config"]["kwargs"]
+                failed_expectations_str += self.create_failed_expectation_text(
+                    expectation_kwargs, expectation_name
+                )
+        return failed_expectations_str
+
+    def create_failed_expectation_text(self, expectation_kwargs, expectation_name):
+        expectation_entity = self.get_failed_expectation_entity(
+            expectation_name, expectation_kwargs
+        )
+        if expectation_entity:
+            return f":x:{expectation_name} ({expectation_entity})\n"
+        return f":x:{expectation_name}\n"
+
+    @staticmethod
+    def get_failed_expectation_entity(
+        expectation_name, expectation_config_kwargs: dict
+    ) -> str:
+        if "expect_table_" in expectation_name:
+            return "Table"
+
+        column_name, column_a, column_b = (
+            expectation_config_kwargs.get("column"),
+            expectation_config_kwargs.get("column_A"),
+            expectation_config_kwargs.get("column_B"),
+        )
+        if column_name:
+            return column_name
+        elif column_a and column_b:
```

Comment:
Could we add one more `if` condition for multicolumn Expectations, looking for a `column_list` kwarg as shown here? https://github.com/great-expectations/great_expectations/blob/d12e9af949181d3a5af59e5e7cc88261b850e83b/great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py#L60

Reply:
thank you @talagluck! I made the required changes and addd a test for the multi-column as well.